pom.xml

```diff
@@ -76,12 +76,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        <dependency>
```

Comment:
You would need to bring in the explicit import of 6.1.12 here. Otherwise its bringing in 6.1.11 which has a new vulnerability.